test/test-cli.js

```diff
@@ -33,7 +33,11 @@ const expectedOneTwo = '\n' +
   '--add-host xyz:1.2.3.4 --add-host abc:5.6.7.8 ' +
   '-h 9c397236341e ' +
   '--expose 4700/tcp --expose 4702/tcp ' +
-  '-e DB_USER=postgres -e "no_proxy=*.local, 169.254/16" ' +
+  '-e \'DB_USER=postgres\' ' +
+  '-e \'no_proxy=*.local, 169.254/16\' ' +
+  '-e \'special_char_env_var1=abc(!)123\' ' +
+  '-e \'special_char_env_var2=abc(\\\')123\' ' +
```

Comment:
Based on my previous recommendation, we would need to change this to the following:

```js
'-e \'special_char_env_var2=abc(\'\\\'\')123\' ' +
```

Reply:
fixed in cf00449